botx/clients/types/http.py

```diff
@@ -24,15 +33,36 @@ class HTTPRequest(BaseModel):
     #: request body.
     json_body: Optional[Dict[str, Any]]
 
+    #: form data.
+    data: Optional[Dict[str, Any]] = None  # noqa: WPS110
+
+    #: file for httpx in {field_name: (file_name, file_content)}.
+    files: Optional[Dict[str, Tuple[str, BytesIO]]] = None  # noqa: WPS234
+
+    #: expected type of response body.
+    expected_type: ExpectedType = ExpectedType.JSON
+
+    #: extra error codes
+    should_process_as_error: List[int] = []
```

Comment:
Do we really need this property? Can we delegate this functionality to `error_handlers`?

Reply:
We decided to keep the current version.